docs/modules/ROOT/pages/config-wizard.adoc

```diff
@@ -63,10 +63,10 @@ Configuration Name: default
 . Paste the Go client sample CURL request link to `Source:` in this screen.
 +
 ```bash
-Source: curl https://api.viridian.hazelcast.com/client_samples/download/...
+Source: curl https://api.cloud.hazelcast.com/client_samples/download/...
 ```
 +
-. Press `[ Submit ]` and it will automatically connect to your Viridian Cloud cluster. Notice that if you didn't specify the configuration name or leave it as default, it will automatically connect to this client unless you specify it with `--config` flag.
+. Press `[ Submit ]` and it will automatically connect to your Hazelcast {hazelcast-cloud} cluster. Notice that if you didn't specify the configuration name or leave it as default, it will automatically connect to this client unless you specify it with `--config` flag.
```

Comment:
Config Wizard feature doesn't exist anymore.
Could you remove `config-wizard.adoc` please?